pkg/phlaredb/schemas/v1/profiles.go

```diff
@@ -28,6 +28,7 @@ var (
 		phlareparquet.NewGroupField("StacktraceID", parquet.Encoded(parquet.Uint(64), &parquet.DeltaBinaryPacked)),
 		phlareparquet.NewGroupField("Value", parquet.Encoded(parquet.Int(64), &parquet.DeltaBinaryPacked)),
 		phlareparquet.NewGroupField("Labels", pprofLabels),
+		phlareparquet.NewGroupField("SpanID", parquet.Encoded(parquet.Uint(64), &parquet.DeltaBinaryPacked)),
```

Comment:
Due to the `DeltaBinaryPacked` is implemented, specifically the bit prefix length of the delta, span IDs are not getting deduplicated.

It may make sense to do delta-encoding/decoding outside parquet, but it only benefits us if there are very few large spans that include substantial share of samples

Reply:
I've decided to not use any encoding or compression as the CPU overhead does not overweight the benefits.

In the future we may want to consider implementing a custom encoding that would handle deltas regardless of the prefix length